File: CPP/Anisul Islam/Number Theory/leap year.cpp
#include <iostream>
using namespace std;
int main()
{
    cout << "Enter the Year: ";
    int year;
    cin >> year;
    (year % 400 == 0) ? cout << "Leap Year" : (year % 4 == 0 && year % 100 != 0) ? cout << "Leap Year"
                                                                                 : cout << "Not Leap Year";
}

File: JavaScript/leap year.js
console.log("Enter The Year");
let year = parseInt(readline());
year % 400 === 0
   ? console.log("Leap Year")
   : year % 4 === 0 && year % 100 !== 0
   ? console.log("Leap Year")
   : console.log("Not Leap Year");
